doc/source/examples/nbsphinx.rst

```diff
@@ -0,0 +1,8 @@
+.. _nbsphinx:
+
+Jupyter notebook - nbsphinx
+===========================
+
```

Comment:
Add a brief description to introduce the notebook?

Reply:
sure, added `This example shows how the Ansys Sphinx Theme renders a Jupyter notebook using the ``nbspinx`` extension.`